index.js

```diff
@@ -301,86 +309,62 @@ function configureBot(bot) {
    * @param itemType
    * @param deliveryThreshold
    */
-  function farmerRoutine(itemType, deliveryThreshold = 10, failureCount = 0) {
-    console.log("Farmer: farmingInProgress=" + farmingInProgress + " , itemType: " + itemType)
+  async function farmerRoutine(itemType, deliveryThreshold = 10, failureCount = 0) {
+    console.log(`Farmer (${failureCount}): farmingInProgress=${farmingInProgress}, itemType: ${itemType}`)
     if (farmingInProgress) {
-
       // do a delivery run
       if (farmingDeliveryRun) {
-        console.log("Farmer: DeliveryRun: Finding a player to deliver to")
+        console.log(`Farmer (${failureCount}): DeliveryRun: Finding a player to deliver to`)
         // find a target player
         const target = Object.entries(bot.players).find((pair) => {
-          console.log("Checking Entity: " + pair[0] + " , " + pair[1].entity?.username)
+          console.log(`Farmer (${failureCount}): Checking Entity: ` + pair[0] + " , " + pair[1].entity?.username)
           // TODO: Need to be able to detect that this is a Human, not another bot
           if (pair[1].entity && pair[1].entity?.username && pair[0] !== bot.entity.username) {
-            console.log("Found Entity")
+            console.log(`Farmer (${failureCount}): Found Entity`)
             return true;
           }
           return false;
         })
         if (target) {
-          console.log("Farmer: DeliveryRun: Trying to deliver " + itemType + " to: " + target[1].entity.username)
-          comeToPlayer(target[1].entity.username, 3).then(async () => {
+          console.log(`Farmer (${failureCount}):  DeliveryRun: Trying to deliver ${itemType} to: ${target[1].entity.username}`)
+          try {
+            await comeToPlayer(target[1].entity.username, 3)
             await bot.lookAt(target[1].entity.position).catch((err) => {
```

Comment:
I'm still a little confused about this - when there is an await, shouldn't it be a try-catch instead of a `.catch(...)`?